### PR TITLE
completed ticket 4 along with styling

### PR DESCRIPTION
**Pull request by hairflicks**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,109 @@
...
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  background-color: #203769;
+}
+
+#reviewCard {
+  width: 75vw;
+  max-width: 500px;
+  height: auto;
```
Comment (callumu21) [inline src/App.css:81]: This is making some of your cards different heights on larger screen sizes when the grid expands. You may want to give this a fixed value to control this.

File: src/api.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import axios from "axios"
```
Comment (callumu21) [inline src/api.js:1]: Really nice!

**Review by callumu21 — CHANGES_REQUESTED** [review]
A really nice start, with some solid styling and responsiveness already. The only thing that needs to be included is:

1. Loading state - for components that depend on an external API call, it's important that we add some loading state and user feedback to let them know that the page is loading. 

I've also left a comment in your stylesheet as it looks like some of your review cards are slightly smaller than others on a larger screen size, but this is something you can fix later.

Once you've added in the loading state, resubmit this PR and we'll take another look 👍

**Review by Pipe-Smoking-Rabbit — APPROVED** [review]
Looks good! 
This is absolutely just a non-core task styling thing but you can get some pretty cool gif images with transparent backgrounds to demonstrate a loading state to a user

**Pull request merged**